src/components/Content/contactUs/ContactForm.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Container, Grid, FormControlLabel, Checkbox } from '@mui/material';
+import { NormalButton } from '@/components/Button/NormalButton';
+import { ControlledTextField } from '@/components/TextField/TextField';
+import { useForm, FormProvider } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import ControlledTypography from '@/components/Typography/Typography';
+import { requiredString } from "@/utils/formSchema";
+
+const schema = z.object({
+  firstName: requiredString("Your firstname is required."),
+  lastName: requiredString("Your lastname is required."),
+  email: requiredString("Your email is required.").email(),
+  message: requiredString("Message is required"),
+  pos: z.boolean(),
+  tms: z.boolean(),
+});
+
+const ContactForm = () => {
+  const methods = useForm({
+    mode: 'all',
+    resolver: zodResolver(schema),
+  });
+
+  const onSubmit = (data: any) => {
+    console.log(data);
+    // Additional steps such as sending the form to a server or processing the data
+  };
+
+  return (
+    <Container style={{ marginTop: '0' }}>
+      <FormProvider {...methods}>
+        <Grid container spacing={2}>
+          <Grid item xs={12}>
+            <ControlledTextField
+              control={methods.control}
+              name="firstName"
+              label="Firstname"
+            />
+          </Grid>
+          <Grid item xs={12} >
+            <ControlledTextField
+              control={methods.control}
```

Comment:
Also when implementing ControlledTextField , there must be required , shouldUnregister